jenkins/opensearch-dashboards/Jenkinsfile

```diff
@@ -42,22 +44,49 @@ pipeline {
                         }
                     }
                 }
-                stage('build-arm64') {
-                    agent {
-                        docker {
-                            label 'Jenkins-Agent-al2-arm64-c6g4xlarge-Docker-Host'
-                            image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdk14-node10.24.1-cypress6.9.1-20211005'
-                            alwaysPull true
+                stage('arm64') {
+                    agent none
+                    stages {
+                        stage('build-arm64') {
+                            agent {
+                                docker {
+                                    label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+                                    image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdk14-node10.24.1-cypress6.9.1-20211005'
+                                    alwaysPull true
+                                }
+                            }
+                            steps {
+                                script {
+                                    build("arm64")
+                                    stash includes : 'artifacts/**', name: 'buildArtifacts'
```

Comment:
Stash's documentation says that it should be used for limited set of files at less than 100mb total and there are suggestions for other methods with https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#stash-stash-some-files-to-be-used-later-in-the-build

As is if this grows we could have strange failures